app-react/src/jsMain/kotlin/com/ustadmobile/view/clazz/detailoverview/ClazzDetailOverviewScreen.kt

```diff
@@ -100,18 +105,31 @@ val ClazzDetailOverviewComponent = FC<ClazzDetailOverviewProps> { props ->
 
                         Stack {
                             direction = responsive(StackDirection.row)
-
-                            if(props.uiState.managePermissionVisible) {
-                                UstadQuickActionButton {
-                                    text = strings[MR.strings.permissions]
-                                    icon = ShieldIcon.create()
-                                    onClick = {
-                                        props.onClickPermissions()
+                            if (props.uiState.managePermissionVisible) {
+                                Stack {
```

Comment:
This is an extra Stack. It should not exist. The buttons are already in a stack. Please remove.